src/Process/3dTilesProcessing.js

```diff
@@ -80,6 +80,10 @@ export function $3dTilesCulling(node, camera) {
 }
 
 export function pre3dTilesUpdate(context, layer) {
+    if (!layer.visible) {
```

Comment:
I'm wondering if we should still clean the layer if the layer is not visible. If I'm not mistaken, tiles that were visible before hiding the layer should not have been marked for deletion, so we avoid removing all the tiles inadvertently.

Reply:
Yeah, this could be an improvment.. I'll keep this in mind for a future PR